__tests__/integration/data_types.test.ts

```diff
@@ -82,53 +80,53 @@ describe('data types', () => {
 
   it('should throw if a value is too large for a FixedString field', async () => {
     const table = await createTableWithFields(client, 'fs FixedString(3)')
-    await expect(
+    await expectAsync(
       client.insert({
         table,
         values: [{ fs: 'foobar' }],
         format: 'JSONEachRow',
       })
-    ).rejects.toEqual(
-      expect.objectContaining({
-        message: expect.stringContaining('Too large value for FixedString(3)'),
+    ).toBeRejectedWith(
+      jasmine.objectContaining({
+        message: jasmine.stringContaining('Too large value for FixedString(3)'),
       })
     )
   })
 
-  it('should work with decimals', async () => {
-    const stream = new Stream.Readable({
-      objectMode: false,
-      read() {
-        //
-      },
-    })
-    const row1 =
-      '1\t1234567.89\t123456789123456.789\t' +
-      '1234567891234567891234567891.1234567891\t' +
-      '12345678912345678912345678911234567891234567891234567891.12345678911234567891\n'
-    const row2 =
-      '2\t12.01\t5000000.405\t1.0000000004\t42.00000000000000013007\n'
-    stream.push(row1)
-    stream.push(row2)
-    stream.push(null)
-    const table = await createTableWithFields(
-      client,
-      'd1 Decimal(9, 2), d2 Decimal(18, 3), ' +
-        'd3 Decimal(38, 10), d4 Decimal(76, 20)'
-    )
-    await client.insert({
-      table,
-      values: stream,
-      format: 'TabSeparated',
-    })
-    const result = await client
-      .query({
-        query: `SELECT * FROM ${table} ORDER BY id ASC`,
-        format: 'TabSeparated',
-      })
-      .then((r) => r.text())
-    expect(result).toEqual(row1 + row2)
-  })
+  // it('should work with decimals', async () => {
```

Comment:
Any actions for this test

Reply:
yes, I need to check this one out, thanks